app/spiffworkflow/messages/MessageHelpers.js

```
@@ -318,34 +318,34 @@
             correlationProperty.correlationPropertyRetrievalExpression.findIndex(
               (retrievalExpr) =>
                 retrievalExpr.messageRef &&
-                retrievalExpr.messageRef.id === messageId
+                retrievalExpr.messageRef.id === messageId,
             );
           if (expr.message_ref == messageId && existingExpressionIndex === -1) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)